internal/command/lint/command.go

```diff
@@ -86,10 +90,22 @@ func Command() *cobra.Command {
 					report.FormatPlain(os.Stderr)
 				}
 			}
+			if jsonPath != "" {
+				report := result.GenerateReport()
+				jsonReport, err := json.Marshal(report)
+				if err != nil {
+					return err
+				}
+				err = ioutil.WriteFile(jsonPath, jsonReport, 0644)
+				if err != nil {
+					return err
+				}
+			}
 			return errors.Errorf("found %d lint errors", len(result.Reports))
 		},
 	}
 	c.Flags().StringVar(&configPath, "config", "", "Path to config file")
 	c.Flags().BoolVarP(&verbose, "verbose", "v", false, "Enable verbose logging")
+	c.Flags().StringVar(&jsonPath, "json", "", "Path to output JSON report file")
```

Comment:
For json output, our plan is to add an output flag instead of taking in a file path. This is already in development, so for simplicity, would you mind removing the JSON report portion of this PR? We're hoping to merge it on our end later this week.

(I will take a look at the code you have right now in `run.go` and make sure our PR offers the same level of usability, so thanks for that!)

Reply:
Removed JSON output. I look forward to see the updated output flag.